File: 프로그래머스/unrated/181841. 꼬리 문자열/꼬리 문자열.js
function solution(str_list, ex) {
    // filter, includes 함수로 str_list 중 ex를 포함하지 않는 것들만 남긴 후,
    // join 함수로 하나의 문자열로 만든다.
    return str_list.filter(el => !el.includes(ex)).join('')
}

File: 프로그래머스/unrated/181841. 꼬리 문자열/꼬리 문자열.py
def solution(str_list, ex):
    # 방법 2 - 리스트 컴프리핸션, join 함수
    return ''.join([string for string in str_list if ex not in string])
    
#     # 방법 1 - for문
#     answer = ''
    
#     for s in str_list:
#         if ex not in s:
#             answer += s
        
#     return answer